src/routes/regions/[region]/+page.svelte

```diff
@@ -22,6 +22,7 @@
 	import type { ComponentEvents, SvelteComponent } from 'svelte';
 	import NdcRange from '$lib/charts/components/NdcRange.svelte';
 	import Sidebar from '$lib/Sidebar.svelte';
+	import NdcDot from '$lib/NdcDot.svelte';
```

Comment:
Shouldn't `NdcDot` be in the same folder as `NdcRange` (`/lib/charts/components/`)?

Reply:
Yes it should